ui/app/AppLayouts/Chat/ContactsColumn/ChannelList.qml

```diff
@@ -83,134 +83,10 @@ Rectangle {
         }
     }
 
-    PopupMenu {
-        property int channelIndex
-        property bool channelMuted
-        property int chatType
-        property string chatName
-        property string chatId
-        property string chatIdenticon
-
+    ChannelContextMenu {
```

Comment:
Since the popup menu in the topbar and here in the channel list is now behaving the same, we could make it so that it is literally the same. They should never be opened at the same time, so we can move this component higher up to a common parent and then when opening it, we just call the same one.